core/http/src/known_media_types.rs

```diff
@@ -18,6 +18,9 @@ macro_rules! known_media_types {
         JPEG (is_jpeg): "JPEG", "image", "jpeg",
         WEBP (is_webp): "WEBP", "image", "webp",
         SVG (is_svg): "SVG", "image", "svg+xml",
+        Icon (is_icon): "Icon", "image", "x-icon",
```

Comment:
I was initially inclined towards `ICO` initially, and am not necessarily opposed to it now—I went that way because all the other ones, the name was `JPEG` for `JPEG` but since the name for `ICO` is `Icon` I went the route of matching `JavaScript`.

Yeah, there is vnd.microsoft.icon—I did look into that one, but it seems that never really got anywhere past Internet Explorer in terms of defaults (and according to [this](https://github.com/paulirish/html5-boilerplate/commit/37b5fec090d00f38de64b591bcddcb205aadf8ee), IE is fine with an `image/x-icon`).

Definitely, welcome to change for ICO depending on where others are at—I would suggest it might be worth concluding on a standard at some point for future additions

Reply:
どうも👍. Firefox 63.0 developer tool implies that *.ico resource without Cointent-Type headers is image/x-icon